be/src/runtime/exec_env.cpp

```diff
@@ -154,6 +154,7 @@ Status ExecEnv::_init(const std::vector<StorePath>& store_paths) {
 
     _num_scan_operators = 0;
     _etl_thread_pool = new PriorityThreadPool("elt", config::etl_thread_pool_size, config::etl_thread_pool_queue_size);
+    _udf_call_pool = new PriorityThreadPool("udf", config::udf_thread_pool_size, config::udf_thread_pool_size);
```

Comment:
must we use `new`， `delete` in exec_env class? could we use std::unique ptr？

Reply:
now all of the xxx_thread_pool were raw pointers. we could refactor them later.